game-core/src/main/java/games/strategy/engine/data/changefactory/units/BombingUnitDamageChange.java

```diff
@@ -0,0 +1,71 @@
+package games.strategy.engine.data.changefactory.units;
+
+import games.strategy.engine.data.Change;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+import java.util.Collection;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.triplea.java.collections.IntegerMap;
+
+/**
+ * A game data change that captures the damage caused to a collection of units by a bombing attack.
+ */
+public class BombingUnitDamageChange extends Change {
+  private static final long serialVersionUID = 7093658184880237574L;
+  private final IntegerMap<String> hits;
+  private final IntegerMap<String> undoHits;
```

Comment:
What is undo hits representing?

Reply:
undoHits represents the inverse of what is happening in this change.  Each change object has an "undo" version of it.

Reply:
The change objects call it "invert".  You can see it being used in the `invert` method.

Reply:
What would you think to rename:
 'hits' -> 'newHits' | 'hitsAfterChange'
'undoHits' -> 'oldHits' | 'hitsBeforeChange' | 'originalHits'

It looks like other change objects use 'new' & 'old' for naming. The 'undoHits' is not the best name, not really clear whether it is the original hits or a delta to compute the original hits.